packages/near-membrane-base/src/blue.ts

```diff
@@ -161,52 +154,99 @@ export function blueProxyFactory(env: MembraneBroker) {
         // istanbul ignore else
         if ('writable' in blueDescriptor) {
             // We are dealing with a value descriptor.
-            const { value } = blueDescriptor;
-            blueDescriptor.value =
-                typeof value === 'function'
-                    ? // We are dealing with a method (optimization).
-                      getBlueFunction(value)
-                    : getBlueValue(value);
+            blueDescriptor.value = getBlueValue(blueDescriptor.value);
         } else {
             // See additional details above.
             // We are dealing with accessors.
             const { get, set } = blueDescriptor;
-            if (typeof get === 'function') {
-                blueDescriptor.get = getBlueFunction(get);
+            if (get !== undefined) {
+                blueDescriptor.get = getBlueValue(get);
             }
-            if (typeof set === 'function') {
-                blueDescriptor.set = getBlueFunction(set);
+            if (set !== undefined) {
+                blueDescriptor.set = getBlueValue(set);
             }
         }
         return blueDescriptor;
     }
 
     function getBlueFunction(redFn: RedFunction): BlueFunction {
-        const blueFn = env.getBlueRef(redFn);
```

Comment:
getBlueRef was basically an abstraction over the weakmap call with nice types and everything. Since the callable boundary will not support such call, we better get rid of that now.